packages/server/jiraImagesHandler.ts

```diff
@@ -0,0 +1,34 @@
+import {HttpRequest, HttpResponse} from 'uWebSockets.js'
+import mime from 'mime-types'
+import sleep from '../client/utils/sleep'
+import uWSAsyncHandler from './graphql/uWSAsyncHandler'
+import {NO_IMAGE_BUFFER} from './utils/atlassian/jiraImages'
+import getRedis from './utils/getRedis'
+
+const getImageFromCache = async (fileName: string, tryAgain: boolean) => {
+  const redis = getRedis()
+  const imageBuffer = await redis.getBuffer(fileName)
+  if (imageBuffer === null || imageBuffer.length === 0) return NO_IMAGE_BUFFER
```

Comment:
we still need a placeholder for a graceful fallback.
lets do the following:
- put a .png of a jira logo in the static images directory
- import it at the top of this file (you may need to add a webpack file-loader to the dev & prod server configs)
- if the image isn't found, return that placeholder (to test in prod, just run `yarn build` then `yarn start`